src/modules/auth/services/idp.service.ts

```diff
@@ -1,16 +1,27 @@
-import { Injectable } from '@nestjs/common';
+import { Injectable, Logger } from '@nestjs/common';
 import { ConfigService } from '@nestjs/config';
+import { get } from 'lodash';
 import queryString from 'query-string';
 
 import { getConfig } from '~config';
 
-import { Idp } from '~shared/auth/auth.types';
+import { SpacesService } from '~modules/spaces/services/spaces.service';
+import { Group } from '~modules/users/types';
+import { Idp, LdapUser } from '~shared/auth/auth.types';
 
 @Injectable()
 export class IdpService {
+	private logger: Logger = new Logger(IdpService.name, { timestamp: true });
 	private idpsWithSpecificLogoutPage = [Idp.HETARCHIEF, Idp.MEEMOO];
 
-	constructor(protected configService: ConfigService) {}
+	public static MEEMOO_ORGANISATION_ID = 'OR-w66976m';
+	public static SHD_ORGANISATION_ID = 'OR-pv6b877';
```

Comment:
should we expose these as environment variables? => i'll ask on the tech sync

Reply:
extracted to env vars